torchgeo/datasets/landsat.py

```diff
@@ -148,6 +152,13 @@ class Landsat1(Landsat):
     default_bands = ('B4', 'B5', 'B6', 'B7')
     rgb_bands = ('B6', 'B5', 'B4')
 
+    wavelengths: ClassVar[dict[str, float]] = {
```

Comment:
In the S12 PR, you also added the alternative names, would that make sense here as well?

Reply:
As far as I know there are no datasets that use the alternative names, so no need yet.